docs/source/index.rst

```diff
@@ -0,0 +1,79 @@
+.. raw:: html
+
+  <h1>Welcome</h1>
+
+  <h2 style="margin-bottom: 10 px">Start Here</h2>
+
+  <div style='column-count: 2; margin-left: 30px;'>
+      <p class="biglink"><a class="biglink" href="{{ pathto("content-quickstart") }}">Jupyter Notebook Quickstart</a><br/>
+         <span class="linkdescr">Try the notebook</span></p>
+      <p class="biglink"><a class="biglink" href="{{ pathto("architecture/content-architecture") }}">Architecture</a><br/>
+         <span class="linkdescr">What is Jupyter?</span></p>
+      <p class="biglink"><a class="biglink" href="{{ pathto("use-cases/content-user") }}">Narratives and Use Cases</a><br/>
+         <span class="linkdescr">Narratives of common deployment scenarios</span></p>
+      <p class="biglink"><a class="biglink" href="{{ pathto("ipython/content-ipython") }}">IPython</a><br/>
+         <span class="linkdescr">An interactive Python kernel and REPL</span></p>
+      <p class="biglink"><a class="biglink" href="{{ pathto("projects/content-projects") }}">Installation, Configuration, and Usage</a><br/>
+         <span class="linkdescr">Documentation for users</span></p>
+     <p class="biglink"><a class="biglink" href="{{ pathto("community/content-community") }}">Community</a><br/>
+         <span class="linkdescr">Sustainability and growth</span></p>
+      <p class="biglink"><a class="biglink" href="{{ pathto("contributor/content-contributor") }}">Contributor Guides</a><br/>
+         <span class="linkdescr">How to contribute to the projects</span></p>
+      <p class="biglink"><a class="biglink" href="{{ pathto("releases/content-releases") }}">Release Notes</a><br/>
+         <span class="linkdescr">New features, upgrades, deprecation notes, and bug fixes</span></p>
+
+      <p class="biglink"><a class="biglink" href="{{ pathto("reference/content-reference") }}">Reference</a><br/>
+         <span class="linkdescr">APIs</span></p>
+      <p class="biglink"><a class="biglink" href="{{ pathto("contents") }}">Contents</a><br/>
+         <span class="linkdescr">Full table of contents</span></p>
+  </div>
+
+  <h2 style="margin-bottom: 10 px">Project Documentation</h2>
+
+  <div style='column-count: 2; margin-left: 30px;'>
```

Comment:
This could also like be simplified with a custom directive. These are just external link, so no need for `path_to`.